tests/Controller/CategoryAdminControllerTest.php

```diff
@@ -261,7 +261,7 @@ public function testTreeAction($context, array $categories): void
 
         if (false !== $context) {
             $contextMock = $this->getContextMock($context);
-            $this->request->query->set('context', $contextMock->getId());
+            $this->request->query->set('context', $contextMock->getId() ?? '');
```

Comment:
```suggestion
            $this->request->query->set('context', $context);
```
The id is the argument of `getContextMock`

Reply:
Way better!